[android] Streamline Google Play metadata (#4367)
organicmaps/organicmaps: android/src/fdroid/play/listings/en-US/full-description.txt

> ‣ Our free app does not track you and does not have ads, and is driven by your help.

Reviewer: ```suggestion
‣ Our free app does not track you, does not have ads, and it needs your support.
```
